nerfstudio/model_components/ray_samplers.py

```diff
@@ -469,14 +470,13 @@ def forward(
             t_max=t_max,
             scene_aabb=self.scene_aabb,
             grid=self.occupancy_grid,
-            # this is a workaround - using density causes crash and damage quality. should be fixed
```

Comment:
Did you find this not to be an issue for real scenes?

Reply:
if anything the existing iNGP code seems to do poorly as it stands - this is what I get after 30k iterations on the bww_entrance dataset:

![image](https://user-images.githubusercontent.com/868618/227739206-9a97578b-04a4-4e98-a630-0ade1c4911e1.png)

(fwiw it seems to actually degrade over time, at 3k iterations it looks like this:)

![image](https://user-images.githubusercontent.com/868618/227739870-0b422f85-393f-4ca9-853f-e44b28164da8.png)

(and then this at 6k iterations):

![image](https://user-images.githubusercontent.com/868618/227739868-ba4f0d2e-0bd7-4181-95a8-10bc20941b3a.png)


And this is what I get after this commit:

![image](https://user-images.githubusercontent.com/868618/227739629-1c7c1315-c461-45aa-8487-1ecb5df3157a.png)

Lmk if there's a more rigorous test suite that I should be running